climpred/tests/test_compute_dims.py

```diff
@@ -214,6 +214,61 @@ def test_PM_fails_probabilistic_member_not_in_dim(
     )
 
 
+@pytest.mark.parametrize('dim', [['member'], ['member', 'init']])
+@pytest.mark.parametrize('metric', ['crpss', 'crpss_es'])
+def test_hindcast_crpss(hindcast_recon_1d_ym, metric, dim):
```

Comment:
Add a brief docstring here. There's many tests so it's helpful to just say you're testing CRPSS here for hindcasts to catch ___ and ___ potential issues. E.g. to check that dims follow ____ standard.